packages/components/button/src/types.ts

```diff
@@ -0,0 +1,2 @@
+export type ButtonType = 'primary' | 'success' | 'warning' | 'danger' | 'info' | 'text' | 'default'
+export type ButtonNativeType = 'button' | 'submit' | 'reset'
```

Comment:
Maybe later we should move this into `tokens/button`.

Reply:
The purpose of this PR is to make the type better, and other more detailed optimizations in another PR #3113